assets/components/lvms/topolvm-controller_rbac.authorization.k8s.io_v1_clusterrole.yaml

```diff
@@ -55,47 +55,3 @@ rules:
   - update
   - patch
   - delete
-- apiGroups:
```

Comment:
This deleted section is now defined separately in the additional clusterrole/binding